pkg/github/projects/project_items.go

```diff
@@ -0,0 +1,318 @@
+package projects
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/araddon/dateparse"
+	"github.com/grafana/github-datasource/pkg/models"
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/pkg/errors"
+	"github.com/shurcooL/githubv4"
+)
+
+// Frames converts the list of Projects to a Grafana DataFrame
+func (p ProjectItemsWithFields) Frames() data.Frames {
+	frame := data.NewFrame(
+		"project_items",
+		data.NewField("id", nil, []string{}),
+		data.NewField("archived", nil, []bool{}),
+		data.NewField("type", nil, []string{}),
+		data.NewField("updated_at", nil, []time.Time{}),
```

Comment:
I believe all the time fields should be `[]*time.Time{}`

Reply:
These times can not be nil.

Reply:
These bool fields can't be nil.